conftest.py

```diff
@@ -55,7 +55,11 @@
 
 @pytest.fixture
 def use_cluster():
-    return os.getenv('SMARTREDIS_TEST_CLUSTER', "").lower() == 'true'
+    if os.getenv('SR_SERVER_TYPE', "") == "":
```

Comment:
this just ignores if the env is not an empty string?

Reply:
The entire fixture is gone now, so this is moot